cardano-node/src/Cardano/Node/Run.hs

```diff
@@ -73,18 +74,20 @@ import qualified Ouroboros.Consensus.Storage.ChainDB as ChainDB
 import           Ouroboros.Consensus.Storage.ImmutableDB (ValidationPolicy (..))
 import           Ouroboros.Consensus.Storage.VolatileDB (BlockValidationPolicy (..))
 
+import           Cardano.Node.Configuration.Socket (SocketOrSocketInfo (..),
+                     gatherConfiguredSockets)
+import           Cardano.Node.Configuration.Topology
+import           Cardano.Node.Handlers.Shutdown
 import           Cardano.Node.Protocol (SomeConsensusProtocol (..), mkConsensusProtocol,
                      renderProtocolInstantiationError)
-import           Cardano.Node.Handlers.Shutdown
-import           Cardano.Node.Configuration.Socket (SocketOrSocketInfo (..), gatherConfiguredSockets)
-import           Cardano.Node.Configuration.Topology
```

Comment:
Our coding style currently allows for `100` characters per line, but we're committing code past this.  Should we extend the line limit to `120`

Reply:
I vote to leave the width at 100 chars

Reply:
As a sucker for code style/formatting, I suggest letting CI enforce the use of `stylish-haskell`, like we do in consensus (https://github.com/input-output-hk/ouroboros-network/blob/7454eaca85e3ffc298e973767316ab2eac2bbe7b/.buildkite/pipeline.yml#L12). This repository is touched by too many different people who are not aware of the style conventions (I'm guilty myself!). Although I think not everybody will agree with making `stylish-haskell` mandatory :grin:.

Reply:
I agree with @mrBliss